abb_irb6600_support/tests/roslaunch_test_irb6600_225_255.xml

```diff
@@ -0,0 +1,23 @@
+<launch>
+  <arg name="ip_str" value="127.0.0.1" />
+
+  <group ns="load_irb6600_225_255__">
+    <include file="$(find abb_irb6600_support)/launch/load_irb6600_225_255.launch"/>
+  </group>
+
+  <group ns="test_irb6600_225_255__">
+    <include file="$(find abb_irb6600_support)/launch/test_irb6600_225_255.launch"/>
+  </group>
+
+  <group ns="robot_interface_download_irb6600_225_255__">
+    <include file="$(find abb_irb6600_support)/launch/robot_interface_download_irb6600_225_255.launch">
+      <arg name="robot_ip" value="$(arg ip_str)" />
+    </include>
+  </group>
+
+  <group ns="robot_state_visualize_irb600_225_255__">
+    <include file="$(find abb_irb6600_support)/launch/robot_state_visualize_irb600_225_255.launch">
```

Comment:
Travis complained: there is no `robot_state_visualize_irb600_225_255.launch` (note: `irb600`).